docs/guidance/map/milestones.md

```diff
@@ -33,6 +33,8 @@ At the point of contract signature, a comparison between `tender/milestones` and
 
 The `dueDate`, `dateMet`, `dateModified` and [`status`](../../schema/codelists.md#milestone-status) fields are used to track the lifecycle of the milestone.
 
+To represent a planned payment, add a `Milestone`, set its `.type` to 'payment' and set its `.value` to the payment's value. Once the milestone's `.status` is met, add a [Transaction](../../schema/reference.md#transaction) to `contracts/implementation/transactions` and set its `.id` and `.value`. For implementation milestones, the transaction can refer back to the milestone using the [transaction related milestones extension](https://extensions.open-contracting.org/en/extensions/transaction_milestones/master/).
```

Comment:
We don't discuss `status` previously, so it's a bit odd to use it in that sentence.

I assume it's understood that the milestone `id`, `status`, `dueDate`, etc. are set and updated as usual. Similarly, we don't need to say which fields to set on the transaction, as it's just the usual.

Alternatively, we can make a bullet list with the full, explicit instructions.

```suggestion
To represent a planned payment, add a `Milestone`, set its `.type` to 'payment' and set its `.value` to the payment's value. Once the milestone is met, add a [Transaction](../../schema/reference.md#transaction) to `contracts/implementation/transactions`. For implementation milestones, the transaction can refer back to the milestone using the [transaction-related milestones extension](https://extensions.open-contracting.org/en/extensions/transaction_milestones/master/).
```